src/main/java/network/brightspots/rcv/Tiebreak.java

```diff
@@ -8,8 +8,11 @@
  */
 
 /*
- * Purpose:
- * Handle tiebreak scenarios based on rules configuration.
+ * Purpose: Handle tiebreak logic based on given rules configuration
+ * Design:  GUI mode uses JavaFX to get the user input if needed.  Cli reads from stdin.
```

Comment:
two spaces after the colon (here and one other occurrence). The rest use one space.